repo-agent/examples/claude-repowatch.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: review.gemini.google.com/v1alpha1
+kind: RepoWatch
+metadata:
+  name: claude
+spec:
+  repoURL: https://github.com/kubernetes/kubernetes
+  githubSecretName: github-pat
+  review:
+    llm:
+      provider: claude
```

Comment:
It would be helpful to add a comment here explaining where the `anthropic-api-key` secret should come from and what key it should contain (e.g., `api-key`).